core/time/zone_spec.rb

```diff
@@ -4,7 +4,7 @@
   platform_is_not :windows do
     it "returns the time zone used for time" do
       with_timezone("America/New_York") do
-        Time.now.zone.should == "EST"
+        Time.now.zone.should =~ /\AE[SD]T\z/
```

Comment:
I thought the zone short name was the same, DST or not while looking at other specs in this file, but they all use a fixed that to avoid this problem.